### PR TITLE
LUSD collateral activation - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: .assets/c3f99efc1151e69157cc7cfbea0481022427ae70.svg
Comment (sakulstra) [inline .assets/c3f99efc1151e69157cc7cfbea0481022427ae70.svg:1]: what is this?

File: src/20230811_AaveV3_Eth_LUSDCollateralActivation/AaveV3_Ethereum_LUSDCollateralActivation_20230811.sol [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3PayloadEthereum, IEngine, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
```
Comment (sendra) [inline src/20230811_AaveV3_Eth_LUSDCollateralActivation/AaveV3_Ethereum_LUSDCollateralActivation_20230811.sol:5]: remove AaveV3Ethereum if not used

File: .assets/c3f99efc1151e69157cc7cfbea0481022427ae70.svg
Comment (marczeller) [reply .assets/c3f99efc1151e69157cc7cfbea0481022427ae70.svg:1]: generated when running tests.

File: src/20230811_AaveV3_Eth_LUSDCollateralActivation/LUSDCollateralActivation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Specification
+
+The proposed risk parameters for LUSD on the Aave V3 ETH pool are as follows:
+
+| Parameter | Value |
+| --- | --- |
+| Loan-to-Value (LTV) | 77% |
+| Liquidation Threshold (LT) | 80% |
+| Liquidation Penalty | 4.5% |
```
Comment (sendra) [inline src/20230811_AaveV3_Eth_LUSDCollateralActivation/LUSDCollateralActivation.md:22]: I would put here the liqProtoclFee also

Comment (marczeller) [reply src/20230811_AaveV3_Eth_LUSDCollateralActivation/LUSDCollateralActivation.md:22]: done

File: src/20230811_AaveV3_Eth_LUSDCollateralActivation/AaveV3_Ethereum_LUSDCollateralActivation_20230811.sol (hunk 1 above)
Comment (marczeller) [reply src/20230811_AaveV3_Eth_LUSDCollateralActivation/AaveV3_Ethereum_LUSDCollateralActivation_20230811.sol]: done

**Review by sendra — APPROVED** [review]

**Pull request merged**